proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -35,3 +35,14 @@ message AuthorizationGrant {
   google.protobuf.Any       authorization = 1 [(cosmos_proto.accepts_interface) = "Authorization"];
   google.protobuf.Timestamp expiration    = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
 }
+
+// DelegateAuthorization allows the grantee to delegate tokens to provided set of validators
+// on behalf of the granter's account.
+message DelegateAuthorization {
+  option (cosmos_proto.implements_interface) = "Authorization";
+
+  repeated string validator_address = 1;
+  cosmos.base.v1beta1.Coin amount = 2
```

Comment:
let's add some godoc for these individual fields. Can we change this `amount` to `max_tokens` or something? This is bit confusing for me actually. May be we can remove `amount` field entirely. I don't see a great utility with that.